py/visualize.py

```diff
@@ -415,7 +415,20 @@ def bubble_chart(responses, poll_ids=None, question_ids=None, xtab1_val="-"):
     # Only subset by xtab1_val if we're not splitting by it.
     xtab_split = xtab1_val != "-"
     if not xtab_split:
-        target_data = target_data[target_data.xtab1_val == xtab1_val]
+        target_data[
```

Comment:
I think you need to reassign `target_data`.

I'd suggest this to avoid hard-coding outside constants:
```python
DROP_POLL_IDS = [
    # NYC democratic primary poll is both a poll of a geographic area and a poll of a demographic that is not comparable to polls on country levels.
    20
]
DROP_QUESTION_IDS = [
    # Willingness to pay higher taxes among supporters of UBI.
    21
]
target_data = target_data[~target_data.poll_id.isin(DROP_POLL_IDS) & ~target_data.question_id.isin(DROP_QUESTION_IDS) & (target_data.xtab1_val == xtab1_val)]
```

Reply:
Yeah, that's definitely better. It might be better to have a `constants.py` file, but I don't think this would need to be accessed anywhere else.